apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/extract/RifFilesProcessorTest.java

```diff
@@ -333,7 +333,7 @@ public void process1CarrierClaimRecord() {
     assertEquals("61026", claimGroup.getCarrierNumber());
     assertEquals("1", claimGroup.getPaymentDenialCode());
     assertEquals(new BigDecimal("199.99"), claimGroup.getPaymentAmount());
-    assertEquals(new BigDecimal("0"), claimGroup.getPrimaryPayerPaidAmount());
+    assertEquals(BigDecimal.ZERO, claimGroup.getPrimaryPayerPaidAmount());
```

Comment:
In theory, yes. However, as noted yesterday, our handling of money, or any decimals for that point, is a mess. As things stand right now, if we did the following: assertEquals(new BigDecimal("0.00"), claimGroup.getPrimaryPayerAmount()); the test fails...it wants a match of zero. Did not want to tackle those types of fixes in this PR since we need to just get the data migrated and code to work with the migrated data. Pretty sure Trotter is going to talk to Karl about cleaning up our decimal handling, etc.